api/mesh/v1alpha1/zone_ingress.proto

```diff
@@ -68,4 +69,7 @@ message ZoneIngress {
   // AvailableService contains tags that represent unique subset of
   // endpoints
   repeated AvailableService availableServices = 3;
+
+  // EnvoyConfiguration provides additional configuration for the Envoy sidecar.
+  EnvoyConfiguration envoy = 4;
```

Comment:
Why do we need that in zone ingress? (I might answer that question myself later)

Reply:
It's because Egress and Ingress also can use delta for configuration exchange